DFE-Digital/academies-academisation-api
# 105758 missing bools (but only found 1)

Adding missing bool onto GET & PUT endpoints as per this ticket:-
https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/105758?McasTsid=26110

## Type of change

## Expected behaviour
ConversionChangeNamePlanned property to be present in GET && PUT endpoint
And, within 'ApplicationSchool' table in sip DB

## Steps to reproduce issue (if relevant)
n/a

## Steps to test this PR
1. Run unit tests to ensure nothing broken
2. Run dotnet ef database update - to ensure new column created
3. run API to ensure new property is available on GET && PUT endpoints

## Prerequisites
n/a
